packages/contentstack-auth/package.json

```diff
@@ -21,15 +21,15 @@
     "test:unit": "mocha --forbid-only \"test/unit/*.test.ts\""
   },
   "dependencies": {
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-command": "^1.2.0",
+    "@contentstack/cli-utilities": "^1.2.0",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K If we use @oclif/core from the utility package we can remove the dependency from package.json right.?

Reply:
@antonyagustine removed @oclif/core dependency from package.json